docroot/modules/humsci/hs_admin/hs_admin.info.yml

```diff
@@ -0,0 +1,6 @@
+name: "HUMSCI Administration"
```

Comment:
Change to `H&S Administration` so it's similarly named with the other custom modules